chcemvediet/locale/sk/LC_MESSAGES/django.po

```diff
@@ -57,29 +57,31 @@ msgstr "Zverejnené infožiadosti zverejniť anonymizovane"
 msgid "accounts:SettingsForm:custom_anonymization:label"
 msgstr "Zadať vlastné slovné spojenia pre anonymizáciu. Štandardne sa anonymizuje meno, priezvisko a adresa žiadateľa."
 
-msgid "accounts:SettingsForm:custom_anonymized_strings:error:empty"
-msgstr "Toto pole je povinné."
-
-msgid "accounts:SettingsForm:custom_anonymized_strings:error:line_too_short"
-msgid_plural "accounts:SettingsForm:custom_anonymized_strings:error:line_too_short {count}"
-msgstr[0] "Slovné spojenie, ktoré sa anonymizuje, musí mať minimálne {count} znak."
-msgstr[1] "Slovné spojenie, ktoré sa anonymizuje, musí mať minimálne {count} znaky."
-msgstr[2] "Slovné spojenie, ktoré sa anonymizuje, musí mať minimálne {count} znakov."
+msgid "accounts:SettingsForm:custom_anonymized_strings:label"
+msgstr "Anonymizované položky"
 
+#, python-brace-format
```

Comment:
@martinmacko47 
Pri dvoch prekladoch sa pridal comment `python-brace-format`. Ostatne zmeny su len zmena poradia niektorych prekladov.

Reply:
`python-brace-format` je pomocka pre programy na editaciu `.po` suborov, ze namiesto `%(foo)s` sa pouziva `{foo}` format na vlozenie premennej. Nam to v principe netreba, kedze `.po` subor editujeme rucne, ale je dobre to tam mat.